pegasus/sites.v3/hourofcode.com/views/faq.haml

```diff
@@ -16,7 +16,7 @@
 
 %details
   %summary= hoc_s(:hoc_faq_how_participate_q)
-  %p!= hoc_s(:hoc_faq_how_participate_a_markdown, markdown: :inline, locals: {howto: resolve_url('/how-to'), campaign_date: campaign_date('start-short')})
+  %p!= hoc_s(:hoc_faq_how_participate_a_markdown, markdown: :inline, locals: {howto: resolve_url('/how-to')})
```

Comment:
This string was causing a Honeybadger error so I'm updating in this PR since this same string will be removed on the Public Officials how-to guide FAQ.